lib/manageiq/password.rb

```diff
@@ -23,58 +22,46 @@ def initialize(str = nil)
       @encStr = encrypt(str)
     end
 
-    def encrypt(str, ver = "v2", key = self.class.keys[ver])
-      value = key.encrypt64(str).delete("\n") unless str.nil? || str.empty?
-      "#{ver}:{#{value}}"
-    end
+    def self.encrypt(str, key = self.key)
+      return str if str.nil?
```

Comment:
so before, blank values are still encoded.
and `""` is encoded. there is a test to back this up.

you're sure you want to make a change like this?

Reply:
thanks - this is all set